daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -280,6 +280,43 @@ async function start() {
 }
 ```
 
+#### Bulk Subscribe to messages
+
+Dapr [supports bulk subscribing](https://v1-10.docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-bulk/) a topic.
```

Comment:
Bulk Subscription is supported and supports two flavors in its API:

- `bulkSubscribeWithDefaultConfig`: Bulk Subscribe to a topic, utilising default values for `maxMessagesCount` and `maxAwaitDurationMs` 
- `bulkSubscribeWithConfig`: Bulk Subscribe to a topic and support setting the `maxMessagesCount` and `maxAwaitDurationMs`

Reply:
Only one flavor supported now, as per further review comments.